src/index.css

```diff
@@ -1,5 +1,15 @@
+html,
 body,
-html {
+#root {
+  height: 100%;
   margin: 0;
   padding: 0;
+  display: flex;
+  flex-direction: column;
+}
+
+#root {
```

Comment:
I'm not sure how this style is impacting the render. I have two concerns here:

- Have you set any id for any element as `root`? If yes, can we please change it? There is already `:root` selector in CSS. Also let's follow BEM convention. Also, I see some inline styling and also a style sheet. We should look at having some standard pattern for this.
- In my opinion, in the case of two similar selectors, only the second one gets applied.

Reply:
Yeah, looking into this. Would def clear this out. Thinking of bringing in a component level style library also like styled-components for taking care of all the styles getting packed up.